exercicios/para-casa/LohanaR/atividadepracasa.py

```diff
@@ -0,0 +1,132 @@
+def main():
+    dataset = {'discentes': []}
+    print("\n---  Olá, seja bem-vinde à Escola do Reprograma!  ---")
+    print("Sistema de informações de discentes")
+
+    while True:
+        cod_opcao = obter_opcao()
+        if cod_opcao == 1:
+            incluir_nova_discente(dataset)
+        elif cod_opcao == 2:
+            consultar_lista_discentes(dataset)
+        elif cod_opcao == 3:
+            consultar_faltas_discente(dataset)
+        elif cod_opcao == 4:
+            consultar_notas_discente(dataset)
+        elif cod_opcao == 5:
+            consultar_status_aprovacao(dataset)
+        elif cod_opcao == 6:
+            print("Encerrando o programa...")
+            encerramento()
+            break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                     "1 - Incluir uma nova aluna\n"
+                                     "2 - Consultar lista de alunas\n"
+                                     "3 - Consultar faltas da aluna\n"
+                                     "4 - Consultar notas da aluna\n"
+                                     "5 - Consultar status de aprovação\n"
+                                     "6 - Sair do sistema\n"
+                                     "Opção: "))
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 6).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+
+    return codigo_opcao
+
+def incluir_nova_discente(dataset):
+    nome = input("Por favor, digite o nome da discente: ")
+    try:
+        faltas = int(input("Por favor, digite o número de faltas: "))
```

Comment:
Aqui, o sistema deveria conseguir incluir a presença da respectiva aula. Futuramente, em outro MVP, se eu quisesse saber em quais aulas a aluna faltou, não haveria essa informação. No requisito funcional havia:
- O sistema deve permitir a inclusão de uma nova aluna com os seguintes dados: nome, sobrenome, turma, notas (uma lista de números), presença (uma lista de booleanos indicando a presença em cada aula) e participação (um número representando o nota de participação da aluna).

**Sugestão:**
```suggestion
    faltas = obter_presença()
    
def obter_presença():
    quantidade_aulas = int(input('Digite quantas aulas houveram: '))
    lista_presença = []

    for contador in range(quantidade_aulas):
        while True:
            entrada = input(f'Presença na aula #{contador +1} (True/False): ')
            if entrada in ['True', 'False']:
                lista_presença.append(eval(entrada))
                break
            else:
                print('Informação inválida. Insira apenas True ou False')
```